api/src/rest/controllers/rooms.js

```diff
@@ -202,5 +202,53 @@ router.put('/:roomId/join/:userId', async (req, res, next) => {
   }
 });
 
+// PUT leave room (unjoin user and room)
+router.put('/:roomId/leave/:userId', isAuthenticated, async (req, res, next) => {
+  const { roomId, userId } = req.params;
+
+  if (!uuidv4RegExp.test(roomId)) {
```

Comment:
You could move this check into a separate function, like `isValidUuid(id: string): boolean` so if you ever want to alter the way you validate your uuid's you can do so from one place.